src/main/java/org/humancellatlas/ingest/migrations/MongoChangeLog.java

```diff
@@ -0,0 +1,19 @@
+package org.humancellatlas.ingest.migrations;
+
+import com.github.mongobee.changeset.ChangeLog;
+import com.github.mongobee.changeset.ChangeSet;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.connection.ServerVersion;
+
+@ChangeLog
+public class MongoChangeLog {
+    @ChangeSet(order = "2019-10-29", id="incrementFeatureCompatibilityVersion", author = "alexie.staffer@ebi.ac.uk", runAlways = true)
```

Comment:
Great Question.
Each Mongo Server supports two feature compatibility versions, the current server version, and the prior version. 
- Server 3.6 supports feature compatibility 3.4 and 3.6
- Server 4.0 supports feature compatibility 4.0 and 3.6

Our current server version is 3.4 and we have 3.2 set as our feature compatibility version. 3.4 is about to leave support and each commit has to get to the production server one at a time so that we don't skip versions.

The above code always runs if required, to bring the feature compatibility up to match the server version, regardless of what the server version is. Which means the migration process is as follows:
1. Commit Auto Upgrade migration
2. Commit server version 3.6
3. Commit server version 4.0
4. Commit server version 4.2

Without the auto update migration, the migration path is as follows:
1. Commit migration to 3.4
2. Commit update to Server Version 3.6
3. Commit migration to 3.6
4. Commit update to Server Version 4.0
5. Commit migration to 4.0
6. Commit update to Server Version 4.2
7. Commit migration to 4.2
There's more coordination needed and it'll take twice as long for each version upgrade too.